client/src/components/auth/authComponents/RegistrationForm.jsx

```diff
@@ -62,9 +56,11 @@ const RegistrationForm = () => {
     password: yup
       .string()
       .required('Please enter your password')
+      .min(8, 'Password must contain at least 8 characters')
+      .max(20, 'Password is too long')
       .matches(
-        /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$/,
-        'Password is not strong enough',
+        /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,20}$/,
+        'Password must be 8 or more characters with a mix of letters and numbers',
```

Comment:
Curious, are we doing any corresponding validation for passwords in the backend as well, if so do we need to change that?

Reply:
As far as i know, I dont think we are doing any password validation when registering a new account in the backend?